src/mgr/PyModuleRegistry.h

```diff
@@ -191,10 +191,12 @@ class PyModuleRegistry
   void unregister_client(std::string_view name, const entity_addrvec_t& addrs)
   {
     auto itp = clients.equal_range(std::string(name));
-    for (auto it = itp.first; it != itp.second; ++it) {
+    for (auto it = itp.first; it != itp.second;) {
       if (it->second == addrs) {
         it = clients.erase(it);
+        continue;
```

Comment:
```suggestion
        return;
```

assuming we don't add multiple clients with identical name **and** addrv.

Reply:
nice. will update.